File: Food2GoWebDio/Modeli/Korisnik.php

<?php


 class Korisnik
{
    private $ime;
    private $prezime;
    private $username;
    private $lozinka;
    private $OIB;
    private $email;
    private $adresa;
    private $mobitel;
    private $brojPokusaja;
    private $status;
    private $aktivacijskiKod;
    private $brojBodova;
    private $tip_korisnika;

    /**
     * Korisnik constructor.
     */
    public function __construct()
    {
    }

    /**
     * @return string
     */
    public function getIme()
    {
        return $this->ime;
    }

    /**
     * @param string $ime
     */
    public function setIme($ime)
    {
        $this->ime = $ime;
    }

    /**
     * @return string
     */
    public function getPrezime()
    {
        return $this->prezime;
    }

    /**
     * @param string $prezime
     */
    public function setPrezime($prezime)
    {
        $this->prezime = $prezime;
    }

    /**
     * @return string
     */
    public function getUsername()
    {
        return $this->username;
    }

    /**
     * @param string $username
     */
    public function setUsername($username)
    {
        $this->username = $username;
    }

    /**
     * @return string
     */
    public function getLozinka()
    {
        return $this->lozinka;
    }

    /**
     * @param string $lozinka
     */
    public function setLozinka($lozinka)
    {
        $this->lozinka = $lozinka;
    }

    /**
     * @return integer
     */
    public function getOIB()
    {
        return $this->OIB;
    }

    /**
     * @param integer $OIB
     */
    public function setOIB($OIB)
    {
        $this->OIB = $OIB;
    }

    /**
     * @return string
     */
    public function getEmail()
    {
        return $this->email;
    }

    /**
     * @param string $email
     */
    public function setEmail($email)
    {
        $this->email = $email;
    }

    /**
     * @return string
     */
    public function getAdresa()
    {
        return $this->adresa;
    }

    /**
     * @param string $adresa
     */
    public function setAdresa($adresa)
    {
        $this->adresa = $adresa;
    }

    /**
     * @return string
     */
    public function getMobitel()
    {
        return $this->mobitel;
    }

    /**
     * @param string $mobitel
     */
    public function setMobitel($mobitel)
    {
        $this->mobitel = $mobitel;
    }

    /**
     * @return integer
     */
    public function getBrojPokusaja()
    {
        return $this->brojPokusaja;
    }

    /**
     * @param integer $brojPokusaja
     */
    public function setBrojPokusaja($brojPokusaja)
    {
        $this->brojPokusaja = $brojPokusaja;
    }

    /**
     * @return bool
     */
    public function getStatus()
    {
        return $this->status;
    }

    /**
     * @param bool $status
     */
    public function setStatus($status)
    {
        $this->status = $status;
    }

    /**
     * @return string
     */
    public function getAktivacijskiKod()
    {
        return $this->aktivacijskiKod;
    }

    /**
     * @param string $aktivacijskiKod
     */
    public function setAktivacijskiKod($aktivacijskiKod)
    {
        $this->aktivacijskiKod = $aktivacijskiKod;
    }

    /**
     * @return integer
     */
    public function getBrojBodova()
    {
        return $this->brojBodova;
    }

    /**
     * @param integer $brojBodova
     */
    public function setBrojBodova($brojBodova)
    {
        $this->brojBodova = $brojBodova;
    }

    /**
     * @return integer
     */
    public function getTipKorisnika()
    {
        return $this->tip_korisnika;
    }

    /**
     * @param mixed $tip_korisnika
     */
    public function setTipKorisnika($tip_korisnika)
    {
        $this->tip_korisnika = $tip_korisnika;
    }

}

File: FOOD2GO/core/src/main/java/hr/foi/air/core/Korisnik.java
package hr.foi.air.core;

public class Korisnik {

    private int id;
    private String ime;
    private String prezime;
    private String username;
    private String lozinka;
    private int status;
    private int brojPokusaja;
    private String adresa;
    private String oib;
    private String email;
    private String mobitel;
    private String aktivacijskiKod;
    private int brojBodova;

    public int getBrojBodova() { return  brojBodova; }

    public void setBrojBodova(int brojBodova) { this.brojBodova = brojBodova; }
    private static Korisnik prijavljeniKorisnik;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getIme() {
        return ime;
    }

    public void setIme(String ime) {
        this.ime = ime;
    }

    public String getPrezime() {
        return prezime;
    }

    public void setPrezime(String prezime) {
        this.prezime = prezime;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getLozinka() {
        return lozinka;
    }

    public void setLozinka(String lozinka) {
        this.lozinka = lozinka;
    }

    public String getOib() {
        return oib;
    }

    public void setOib(String oib) {
        this.oib = oib;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getAdresa() {
        return adresa;
    }

    public void setAdresa(String adresa) {
        this.adresa = adresa;
    }

    public String getMobitel() {
        return mobitel;
    }

    public void setMobitel(String mobitel) {
        this.mobitel = mobitel;
    }

    public String getAktivacijskiKod() {
        return aktivacijskiKod;
    }

    public void setAktivacijskiKod(String aktivacijskiKod) {
        this.aktivacijskiKod = aktivacijskiKod;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

    public int getBrojPokusaja() {
        return brojPokusaja;
    }

    public void setBrojPokusaja(int brojPokusaja) {
        this.brojPokusaja = brojPokusaja;
    }

     public static Korisnik getPrijavljeniKorisnik() {
        return prijavljeniKorisnik;
    }

    public static void setPrijavljeniKorisnik(Korisnik prijavljeniKorisnik) {
        Korisnik.prijavljeniKorisnik = prijavljeniKorisnik;
    }

    public String vratiImeiPrezime(){
        return this.ime+" "+this.prezime;
    }

    public Korisnik() {
    }

    public Korisnik(String ime, String prezime, String username, String lozinka, String oib, String email, String adresa, String mobitel, String aktivacijskiKod) {
        this.ime = ime;
        this.prezime = prezime;
        this.username = username;
        this.lozinka = lozinka;
        this.oib = oib;
        this.email = email;
        this.adresa = adresa;
        this.mobitel = mobitel;
        this.aktivacijskiKod = aktivacijskiKod;
    }

    public Korisnik(String username, String lozinka) {
        this.username = username;
        this.lozinka = lozinka;
    }
}